sparkmagic/sparkmagic/controllerwidget/manageendpointwidget.py

```diff
@@ -30,7 +30,13 @@ def get_existing_endpoint_widgets(self):
 
             # Endpoints
             for url, endpoint in self.endpoints.items():
-                endpoint_widgets.append(self.get_endpoint_widget(url, endpoint))
+                try:
+                    endpoint_widgets.append(self.get_endpoint_widget(url, endpoint))
+                except HttpClientException:
+                    # If we can't reach one of the default endpoints, just skip over it
+                    # silently
+                    if not endpoint.is_default:
```

Comment:
I think logging would be a good idea for the case when it's silently ommited from the list and the user files an issue saying it's being ommited. We will then be able to look at the log file and explain what's going on. Logging shouldn't interfere with what the UX is, since it's just writting to a file/logging system.